libretro_core_options.h

```diff
@@ -124,7 +124,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(dither_mode),
       "Dithering Pattern",
       NULL,
-      "Set dithering pattern configuration. '1x (Native)' emulates native low resolution dithering used by original hardware to smooth out color banding artifacts visible at native color depth. 'Internal Resolution' scales dithering granularity to the configured internal resolution for cleaner results. Recommended to be disabled when running at 32 bpp color depth. Note: On Vulkan, enabling this will force downsampling to native color depth, while disabling will automatically enable output at higher color depth.",
+      "Choose dithering pattern configuration. '1x (Native)' emulates native low resolution dithering used by original hardware to smooth out color banding artifacts visible at native color depth. 'Internal Resolution' scales dithering granularity to the configured internal resolution for cleaner results. Recommended to be disabled when running at 32 bpp color depth. Note: On Vulkan, enabling this will force downsampling to native color depth, while disabling will automatically enable output at higher color depth.",
```

Comment:
This is an overall warning of a probable lack of articles in some of the phrases. Examples in here: "emulates the native low resolution dithering"...